app/src/main/java/com/github/sdpteam15/polyevents/database/FirestoreDatabaseProvider.kt

```diff
@@ -162,36 +153,34 @@ object FirestoreDatabaseProvider : DatabaseInterface {
     override fun updateUserInformation(
         newValues: java.util.HashMap<String, String>,
         uid: String,
-        userAccess: UserInterface
+        userAccess: UserEntity?
     ): Observable<Boolean> = thenDoSet(
         firestore!!.collection(USER_COLLECTION)
             .document(uid)
             .update(newValues as Map<String, Any>)
     )
 
-
+    // TODO: First connection is when registering? Because this way we are overwriting data
```

Comment:
No first connection is called only when the user is not already in the database, there are checks in the login fragment. You can safely remove this comment :)

Reply:
Yes sorry just a sanity check. Will remove it.